docs/DeveloperGuide.md

```diff
@@ -569,13 +569,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                      | find food stall by matching a word in the name                                                                               | find the exact food stall I am interested in                                                                                                           | Yes                                               |
 | `*`      | user                      | find food stall by approximate name                                                                                          | find the exact food stall I am interested in even when I’m not very sure about the stall name                                                          | No                                                |
 | `*`      | user                      | tag a food stall with a tag                                                                                                  | categorize food stalls effectively                                                                                                                     | Yes                                               |
-| `*`      | user                      | list out food place according to given tag                                                                                   | get an overview of the food place with the tag I am interested in                                                                                      | Yes                                               |
+| `*`      | user                      | list out food stalls that contain a given tag                                                                                | find all food stalls that contain the tag I am Int                                                                                                     | Yes                                               |
```

Comment:
Perhaps you left out something? I am Int does not sound right.

Reply:
thanks for pointing out, fixed